### PR TITLE
Update OZW docs for new config parameter type and provide example

### PR DESCRIPTION
**Pull request by raman325**
## Proposed change
My parent PR adds support for a new parameter type for the `set_config_parameter` service, so this PR updates the docs to instruct users on how to use the service for that type.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/40527
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ozw.markdown [hunk 1]
```diff
@@ -98,7 +98,21 @@ LED colors on switches.
...
 | `node_id`              | yes      | Node id of the device to set configuration parameter to (integer).                                              |
 | `parameter`            | yes      | Parameter number to set (integer).                                                                              |
-| `value`                | yes      | Value to set for parameter. (String or integer value for list, string for bool parameters, integer for others). |
+| `value`                | yes      | Value to set for parameter. (String or integer value for list, string or boolean for bool parameters, dictionary for bitset parameters (see example below), integer for others). |
+
+
+#### Example BitSet service call
+The dictionary passed as the value should be of the form <BIT POSITION OR LABEL>: <1 for on or 0 for off>
+
+Here is an example of what to send to the service for a BitSet parameter:
```
Comment (frenck) [inline source/_integrations/ozw.markdown]:
```suggestion
#### Example BitSet service call

The dictionary passed as the value should be of the form <BIT POSITION OR LABEL>: <1 for on or 0 for off>

Here is an example of what to send to the service for a BitSet parameter:

```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by raman325** [comment]
Thanks @frenck I am going to close this for now and re-open it against a new PR. Turns out I did not implement the change documented here correctly so will have to submit a PR to fix

**Comment by raman325** [comment]
This change was reintroduced into the parent PR but the schema is different. PR updated accordingly

**Review by frenck — APPROVED** [review]
Thanks, @raman325 👍

**Pull request merged**